doc/examples/template.json

```diff
@@ -7,17 +7,24 @@
 	 * to set the timer's period to 0
 	 */
 	"tasks" : {
+		/* no 'loop' value at task level means loop indefinitely (-1) */
 		"thread0" : {
-			"instance" : 1,
-			"loop" : -1,
-			"run" :   10000,
-			"sleep" : 0,
+			"instance" : 1,  /* number of threads to be created */
+			"loop" : -1,  /* loop forever (see 'duration' below) */
+
+			"run" :   10000,  /* when woken up,
+				execute 10000 / pLoad loops (note: rounding) */
+
+			"sleep" : 0, /* after running, do not sleep, (then
+				go to sleep until timer fires) */
```

Comment:
In fact this event doesn't generate anything because sleep is 0 and is somewhat useless.

Reply:
ok, changing that accordingly